project-koku/koku
# Adding correct rbac perms

Fixes -- OCI rbac perm missing

Changes proposed in this PR:
* OCI rbac perm missing


Testing Instructions:
1. check you can access OCI apis correctly with rbac permissions set

---
Additional Context:
